.gitignore

```
@@ -1,4 +1,5 @@
 _site/
 .DS_Store
 
-assets/scaled_thumbnails/
\ No newline at end of file
+assets/scaled_thumbnails/
+.Rproj.user
```

@solanoamy would you do me a favor and also add .RData, .Rhistory, .Rproj files, and the rosm.cache and sources folders to this list...
```suggestion
.Rproj.user
.RData
.Rhistory
*.Rproj

rosm.cache/
sources/
```
and then delete those files and folders from the repository? If you don't want to delete them, I'd move them to somewhere outside of your repository, commit the change, and then move them back in.